book/src/intro.md

```diff
@@ -81,6 +82,20 @@ Those principles are:
 
 [up]: https://en.wikipedia.org/wiki/Unix_philosophy
 
+But with all the [great "blocks" in the Rust gamedev ecosystem][awg], why not glue your own engine together?
+
+Glueing blocks together involves a lot more than one might think. While Amethyst is quite modular by itself, we have
+a big advantage over standalone libraries: All of our crates have a common basis, [Specs][specs] (an ECS framework).
+Due to Specs' nature we have a low-cost abstraction that needs close to no glue between the crates. If another crate
```

Comment:
Due to Specs' nature => Due to the nature of Specs
Also I think "needs" should be "means", right?